scripts/next_extension_version.mjs

```
@@ -0,0 +1,93 @@
+    throw new Error('Could not find any release tags (`x.y.z` or `insider/x.y.z`) to derive the next version from.')
+  }
+  return versions[0]
+}
+
+function parseStablePrismaVersion(prismaVersion) {
+  const parsed = semVer.parse(prismaVersion)
+  if (parsed === null) {
+    throw new Error(`Invalid Prisma CLI version '${prismaVersion}'. Expected a semantic version such as 7.9.0.`)
+  }
+  if (parsed.prerelease.length > 0) {
+    throw new Error(`Prisma CLI version '${prismaVersion}' is a prerelease and can not be released on the stable channel.`)
```

ESLint

[error] Replace ``Prisma·CLI·version·'${prismaVersion}'·is·a·prerelease·and·can·not·be·released·on·the·stable·channel.`` with `⏎······`Prisma·CLI·version·'${prismaVersion}'·is·a·prerelease·and·can·not·be·released·on·the·stable·channel.`,⏎····`

(prettier/prettier)